### DISCUSSION TITLE
dietpi-config option to set the DMC governor in Orange Pi 5

### DISCUSSION THREAD

**Discussion by vanfanel** [MichaIng/DietPi#6326] [Software Request]
Hello there!

Currently there's an option in `dietpi-config` to set the CPU governor under `performance`, but we're missing an option to set the DMC (Dynamic Memory Controller) governor.

To make things easier, I will note that the DMC governor can be easily set to `performance` with:
`echo performance | tee /sys/class/devfreq/dmc/governor`

...so the option could be pretty trivial to add, I hope!

**Comment by MichaIng** [comment]
Many thanks for your request.

Interesting, memory controller scheduling. I wonder whether this is unique on Rockchip's RK3588 vendor kernel or exists on mainline Linux as well. Will check on NanoPi R6S/R6C.

Yes is indeed easy to implement. What is the default?

**Comment by vanfanel** [comment]
@MichaIng Available values are:
```

root@DietPi:~# cat /sys/class/devfreq/dmc/available_governors
rknpu_ondemand dmc_ondemand userspace powersave performance simple_ondemand
```

And the default value is:
`
dmc_ondemand`
